dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/MultiSwitch.mo

```diff
@@ -14,7 +14,7 @@ within Dynawo.NonElectrical.Blocks.NonLinear;
 
 block MultiSwitch "Switch between N Real signals"
 
-  parameter Integer nu(min=0) = 0 "Number of input connections" annotation(
+  parameter Integer nu(min = 1) "Number of input connections" annotation(
```

Comment:
It should be kept as it was, nu(min=0) = 0 as proposed in Modelica library. It could be important for compiler behind it

Reply:
Agreed